string fromToUser
---
int64 status
string info

